### DISCUSSION TITLE
I get an error when running the genSources task for the 1.19 Fabric example mod in VSCode

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#2730] [Mod Dev Support] [answered]
I get an error when running `./gradlew genSources --stacktrace` for the Fabric example mod in VSCode
https://gist.github.com/TechPro424/b9add985a96f7886a20f7b455de3b5e9
pls help me out

**Comment by warjort** [comment, marked as answer]
````
Caused by: java.nio.file.FileSystemException: C:\Users\User\Coding\Item Blacklist Mod\.gradle\loom-cache\1.19.2\net.fabricmc.yarn.1_19_2.1.19.2+build.28-v2\minecraft-project-@-merged-named.jar: 
The process cannot access the file because it is being used by another process
        at net.fabricmc.loom.task.GenerateSourcesTask$DecompileAction.doDecompile(GenerateSourcesTask.java:287)
        ... 23 more
````
Something still holds a file system lock on the generated/deobfuscated minecraft jar.
Try exiting your IDE and using task manager to check for still running java processes.

**Comment by TechPro424** [comment]
There were no java processes when I exited the IDE, but when I re-opened the IDE and tried again it worked
(reopening hadn't worked yesterday at all, though, that's weird)

Thanks for your time!
